src/Panel/PageCreateDialog.php

```diff
@@ -301,13 +305,20 @@ public function validate(array $input, string $status = 'draft'): bool
 
 	public function value(): array
 	{
-		return [
+		$value = [
 			'parent'   => $this->parentId,
 			'section'  => $this->sectionId,
 			'slug'     => $this->slug ?? '',
 			'template' => $this->template,
 			'title'    => $this->title ?? '',
 			'view'     => $this->viewId,
 		];
+
+		// add default values for custom fields
+		foreach ($this->customFields() as $name => $field) {
+			$value[$name] = $field['default'] ?? null;
```

Comment:
I think this is risky because some fields don't really like null as value. I would probably create an if clause and only add it to the array if it exists. 

```suggestion
if (isset($field['default']) === true) {
  $value[$name] = $field['default'];
}
```

Reply:
Good idea

Reply:
Added the change